doc/manual/manual.tex

```diff
@@ -5023,6 +5023,34 @@ \subsubsection{Prescribed internal velocity constraints}
     \label{fig:prescribed-velocity}
 \end{figure}
 
+\subsubsection{Artificial viscosity smoothing}
+\label{sec:artificial-viscosity-smoothing}
+\textit{This section was contributed by Ryan Grove}\\
+
+Standard finite element discretizations of advection-diffusion equations introduce unphysical oscillations around steep gradients. Therefore, stabilization must be added to the discrete formulation to obtain correct solutions. In ASPECT, we use the Entropy Viscosity scheme developed by Guermond et al. in the paper \cite{guer11}. In this scheme, an artificial viscosity is calculated on every cell in and used to try to combat these oscillations that cause unwanted overshoot and undershoot.  More information about how ASPECT does this is located at \url{https://dealii.org/developer/doxygen/deal.II/step_31.html}.  \\
```

Comment:
please use `\aspect{}` instead of `ASPECT`. This way, the formatting of the name is consistent everywhere.